arches_lingo/etl_modules/migrate_to_lingo.py

```diff
@@ -442,10 +442,22 @@ def init_relationships(
         part_of_scheme_nodegroup = NodeGroup.objects.get(
             nodegroupid=CONCEPTS_PART_OF_SCHEME_NODEGROUP_ID
         )
+        concepts_with_scheme = {}
         for concept in concept_hierarchy:
             root_list = str(concept["root_list"])
             resourceinstanceid = concept["concept"]
 
+            concept_has_part_of_scheme = resourceinstanceid in concepts_with_scheme
+            existing_scheme = concepts_with_scheme.get(resourceinstanceid)
+            if not concept_has_part_of_scheme:
+                concepts_with_scheme[resourceinstanceid] = root_list
+            elif concept_has_part_of_scheme and existing_scheme == root_list:
+                continue
+            elif concept_has_part_of_scheme and existing_scheme != root_list:
+                raise Exception(
```

Comment:
```suggestion
                raise CommandError(
```

Reply:
Updated to follow pattern in [single_csv here](https://github.com/archesproject/arches/blob/ab1e5bf7038b304cdd10f93bd1f4a27829022549/arches/app/etl_modules/import_single_csv.py#L192C1-L198C14). Will need to think about how to handle errors like this when we update this ETL module to be run via CLI, re. https://github.com/archesproject/arches/pull/10519